### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Kakalanp**
Adding visual part of the calculator using a CSS modules.

### CODE REVIEW

File: src/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```
Comment (rotimiazeez) [inline src/setupTests.js]: - [x] Please note that all the boilerplate that got created by the CRA (texts, images, and styles) ought to have gotten deleted.💪🏽 For example:
- App.test.js should be deleted.
- setupTests.js
- reportWebVitals.js
- The react logo images on /public
- If there is CSS code you didn't write, please remove it

File: src/components/Calculator.module.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+.calculator__button__orange {
+  background-color: #f39148;
+  text-align: center;
+}
+
+.calculator__button__zero {
+  grid-column: 1 / 3;
+}
```
Comment (rotimiazeez) [inline src/components/Calculator.module.css]: - [x] Kindly make sure your design match that of the guideline
Your Implementation | Guideline Design
:---:|:---:
screenshot (29) | screenshot (28)

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Kakalanp 👋, 
## Good job so far!
### Summay 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.module.css (hunk 2 above)
Comment (Kakalanp) [reply src/components/Calculator.module.css]: My reasoning behind was that the calculator may be adaptable to different sizes in a real scenario, should I constrain it's size either way? 🤔

File: src/setupTests.js (hunk 1 above)
Comment (Kakalanp) [reply src/setupTests.js]: I wasn't sure if those needed to be deleted too, I'll do right away! 😉

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @Kakalanp, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**